Doc/whatsnew/3.11.rst

```diff
@@ -1877,8 +1877,8 @@ and will be removed in Python 3.12.
 C APIs pending removal are
 :ref:`listed separately <whatsnew311-c-api-pending-removal>`.
 
-* The :mod:`asynchat` module
-* The :mod:`asyncore` module
+* The :mod:`!asynchat` module
```

Comment:
There's a couple other warnings in this paragraph about presumably-removed things that you could maybe fix as well while we're here?